.github/workflows/java8_integration_tests_webui.yml

```diff
@@ -71,6 +72,7 @@ jobs:
           retention-days: 7
       - name: "Upload Code Coverage Report"
         uses: codecov/codecov-action@v1
+        if: always()
```

Comment:
if generating fails, then I believe that the uploading task will fail since it won't be able to find a coverage report

Reply:
Unfortunately, when tests fail, generating can still succeed and in that case, you want to upload the report.

Reply:
yes, but for failed tests, it is possible to miss coverage which makes the report not so helpful

```
test() {
doSomething() // covers 50% of lines
assert(condition) // <--- test fails here
doSomethingElse() // covers remaining 50%
assert(condition2)
}
```

I guess if you still want to check coverage for a section of code where the test passed, the upload can still be useful, but the overall numbers will probably still differ in a significant manner

Reply:
Obviously the report isn't full. But what's more important is that the codecov knows that there is a failed test in the coverage. That's important because or else, it thinks failed builds are successful builds that are treated as baseline for coverage.